impl/src/main/java/com/okta/sdk/impl/resource/identity/provider/DefaultGoogleIdentityProviderBuilder.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2020-Present Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.okta.sdk.impl.resource.identity.provider;
+
+import com.okta.sdk.client.Client;
+import com.okta.sdk.resource.identity.provider.IdentityProvider;
+import com.okta.sdk.resource.identity.provider.IdentityProviderCredentials;
+import com.okta.sdk.resource.identity.provider.IdentityProviderCredentialsClient;
+import com.okta.sdk.resource.identity.provider.Protocol;
+import com.okta.sdk.resource.identity.provider.Provisioning;
+import com.okta.sdk.resource.identity.provider.ProvisioningConditions;
+import com.okta.sdk.resource.identity.provider.ProvisioningDeprovisionedCondition;
+import com.okta.sdk.resource.identity.provider.ProvisioningGroups;
+import com.okta.sdk.resource.identity.provider.ProvisioningSuspendedCondition;
+import com.okta.sdk.resource.policy.IdentityProviderPolicy;
+import com.okta.sdk.resource.policy.PolicyAccountLink;
+import com.okta.sdk.resource.policy.PolicySubject;
+import com.okta.sdk.resource.policy.PolicyUserNameTemplate;
+
+public class DefaultGoogleIdentityProviderBuilder extends DefaultIdentityProviderBuilder {
```

Comment:
nit: this probably doesn't need to change for the release, as it is in the `impl` project. but these providers could probably have the `Default` removed from the beginning, as there is no longer an interface with same name

Reply:
done